onnxmltools/convert/coreml/operator_converters/neural_network/Activation.py

```diff
@@ -46,6 +46,7 @@ def convert_activation(scope, operator, container):
     else:
         if activation_type == 'thresholdedReLU':
             op_type = 'ThresholdedRelu'
+            attrs['op_version'] = 10
```

Comment:
What if the targeted `op_version < 10`? Shouldn't it be another `apply_...`.

Reply:
ThresholdedReLu graduated to a full op from an experimental op -- it doesn't have an official implementation in the ONNX spec prior to Opset 10. Should we throw an error if container.target_opset < 10?